src/components/VMenu/VMenu.js

```diff
@@ -189,9 +189,7 @@ export default {
   render (h) {
     const data = {
       staticClass: 'v-menu',
-      style: {
-        display: this.fullWidth ? 'block' : 'inline-block'
-      },
+      class: { 'v-menu--block': this.fullWidth && this.$slots.activator },
```

Comment:
as per [Kael's comment](https://github.com/vuetifyjs/vuetify/pull/4577#discussion_r201750297) it should be the other way round - `v-menu` should be `block` by default, and if `!this.fullWidth && this.$slots.activator` it should have `v-menu--inline` class (which needs to be added in .styl)

Reply:
Ok, perfect. I'm now testing it with the playground and I see that v-menu--block does not exists. I first saw it on the _menus.styl